eo-runtime/src/test/eo/org/eolang/txt/text-tests.eo

```diff
@@ -39,3 +39,24 @@
       text ".."
       * "foo" "друг" "bar"
     $.equal-to "foo..друг..bar"
+
+[] > joins-array-2
+  assert-that > @
+    joined.
+      text ", "
+      * "Привет" "мир!"
+    $.equal-to "Привет, мир!"
+
+[] > joins-array-3
+  assert-that > @
+    joined.
+      text ""
+      *
+    $.equal-to ""
+
+[] > joins-array-4
+  assert-that > @
+    joined.
+      text ""
+      * "foo" "друг" "bar"
+    $.equal-to "fooдругbar"
```

Comment:
@EugeneDar let's add one more test with nested `joined` objects. e.g.
```
[] > nested-joins
  assert-that > @
    joined.
      joined.
        text "&"
        * " " " "
      * "start" "end"
    $.equal-to "start & end"
```

Reply:
@Graur maybe it should look like this:
```
[] > nested-joins
  assert-that > @
    joined.
      text
        joined.
          text "&"
          * " " " "
      * "start" "end"
    $.equal-to "start & end"
```